components/automate-ui/src/app/page-components/search-bar/search-bar.component.ts

```diff
@@ -12,30 +12,30 @@ import {
 import { Subject, Observable, of as observableOf } from 'rxjs';
 import { List } from 'immutable';
 import { clamp, compact } from 'lodash';
-import { Chicklet } from '../../types/types';
+import { SearchBarCategoryItem, Chicklet, SuggestionItem } from '../../types/types';
```

Comment:
The below changes add static/providedValues to the search bar and allows categories to not allow wildcard filters.